standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/jdbc/functions/MinOpenTxnIdWaterMarkFunction.java

```diff
@@ -61,7 +61,7 @@ public Long execute(MultiDataSourceJdbcResource jdbcResource) throws MetaExcepti
           return id;
         }));
     long lowWaterMark = jdbcResource.execute(new OpenTxnTimeoutLowBoundaryTxnIdHandler(openTxnTimeOutMillis));
-    LOG.debug("MinOpenTxnIdWaterMark calculated with minOpenTxn {}, lowWaterMark {}", minOpenTxn, lowWaterMark);
+    LOG.info("MinOpenTxnIdWaterMark calculated with minOpenTxn {}, lowWaterMark {}", minOpenTxn, lowWaterMark);
```

Comment:
i don't think we should be changing to info here, it's a debug information

Reply:
Rolled back